webui/helpers_test.go

```diff
@@ -0,0 +1,141 @@
+package webui
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/contribsys/faktory/client"
+)
+
+type testJob struct {
+	name     string
+	job      *client.Job
+	expected string
+}
+
+func makeActiveJob(jobType string) *client.Job {
+	var payload = []byte(fmt.Sprintf(`
```

Comment:
I got these payloads by sticking this in my `displayJobType` function:

```golang
	out, err := json.Marshal(j)
	if err == nil {
		util.Infof("Job: %s", out)
	}
```

So this should be exactly what the Web UI receives at render time. I parametrized certain things as made sense. I left the IDs and timestamps and whatnot alone since there's nothing particularly interesting in those.